src/components/NextThreeShows.js

```diff
@@ -9,23 +9,29 @@ import { showType } from "../lib/types.js";
 import styles from "../styles/NextThreeShows.module.css";
 import ShowSnippet from "../components/ShowSnippet.js"
 
-export default function NextThreeShows({ shows, handleClick }){
+export default function NextThreeShows({ shows, handleClick, setCurrentPage }){
 
   const nextThree = shows.map((show) => 
     <li className={styles.showListItem} key={show.id} onClick={() => handleClick(show)} data-testid="show snippet">
       <ShowSnippet show={show}/>
     </li>
   );
 
+  function changePage() {
+    setCurrentPage("Schedule"); 
+  }
+
   return (
     <div className={styles.nextThreeShows}>
       <div className={styles.header}>Today&apos;s next three shows:</div>
       <ul className={styles.showList}>{nextThree}</ul>
       {(nextThree.length < 3) && <div className={styles.message}>That&apos;s all for today!</div>}
+      <input className={styles.button} type="button" value="See Full Schedule" onClick={changePage}/>
```

Comment:
I think this can be simplified even more by just calling `setCurrentPage` in the `onClick`: 

`onClick={() => setCurrentPage("Schedule")}`

Then we don't even need `changePage`

Reply:
lol thanks. didn't realise i needed the '() =>' thingy to make it work